package.json

```diff
@@ -51,8 +51,8 @@
     "yeoman-test": "^6.0.0"
   },
   "dependencies": {
-    "@adobe/generator-add-action-asset-compute": "^2",
-    "@adobe/generator-app-common-lib": "^2",
+    "@adobe/generator-add-action-asset-compute": "^3",
```

Comment:
@purplecabbage did we release a v3-still see [2.0.1](https://github.com/adobe/generator-add-action-asset-compute/blob/main/package.json#L3)? or after [60](https://github.com/adobe/generator-add-action-asset-compute/pull/60), we plan to bump the major version to 3 and then release this?

Reply:
yeah, this depends on a new version of add-action